cmd/bundle/generate/generate_test.go

```diff
@@ -215,6 +216,73 @@ func TestGenerateJobCommand(t *testing.T) {
 	require.Equal(t, "# Databricks notebook source\nNotebook content", string(data))
 }
 
+func TestGenerateJobWithGitSource(t *testing.T) {
```

Comment:
Since we have the acceptance test, we can get rid of this?

Reply:
+1 would be great to delete redundant tests